client/src/components/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState, } from 'react';
+import styles from './ResetPassword.module.scss';
+import TextField from '../TextField/TextField.tsx';
+import { Button } from '@mui/material';
```

Comment:
The custom button component for our project is live. Feel free to update it in another PR to avoid any conflicts for now.